__tests__/components/Header-test.tsx

```diff
@@ -16,7 +16,7 @@ describe("Header", () => {
     expect(text.text()).to.equal("Test Header!");
   });
   it("should render a logout button", () => {
-    let logout = wrapper.find(".navbar-btn");
+    let logout = wrapper.find(".header-btn");
```

Comment:
Eventually, this should be more specific in case you have more buttons in the header.